website/content/docs/sync/index.mdx

```diff
@@ -127,6 +127,24 @@ Granularity can be updated. The new granularity only affects secrets newly assoc
 not modify the previously synced secrets. It is possible to update an association to force a recreate operation.
 The secret synced with the old granularity will be deleted and new secrets will be synced according to the new granularity.
 
+
+## Reconciliation
+
+Vault Secrets Sync is designed to automatically recover from transient failures
+in two ways: operation retries and reconciliation scans.
+
+Operation retries happen when a sync operation fails. Vault automatically
+retries the operation with exponential backoff. Operation retries help in
+situations where your network becomes unreliable or overwhelmed.
```

Comment:
I think the doc rarely uses the 2nd person point of view with you/your. I think it is more typical to use a 3rd person voice so in this case we'd address just "the network" or "Vault's network" not "your network".